fbw-a380x/src/systems/instruments/src/ND/NDSimvarPublisher.tsx

```diff
@@ -34,11 +36,13 @@ export enum NDVars {
   ilsCourse = 'L:A32NX_FM_LS_COURSE',
   selectedHeading = 'L:A32NX_FCU_HEADING_SELECTED',
   showSelectedHeading = 'L:A320_FCU_SHOW_SELECTED_HEADING',
-  pposLat = 'PLANE LATITUDE', // TODO replace with fm position
-  pposLong = 'PLANE LONGITUDE', // TODO replace with fm position
+  pposLat = 'PLANE LATITUDE', // FIXME replace with fm position
+  pposLong = 'PLANE LONGITUDE', // FIXME replace with fm position
+  pposAlt = 'PLANE ALTITUDE', // FIXME replace with fm position
```

Comment:
This seems dubious? We already have the ADR altitude topics.

Reply:
I thought I would keep it consistent with the pposLat&Lon, but I saw that those were also not used anymore. So I removed them (the ADR lat&lon are also used in the ND), and replaced pposAlt with ADR's baroCorrectedAltitude